src/test/java/com/datasonnet/HeaderTest.java

```diff
@@ -141,4 +142,28 @@ void testUnterminatedHeaderFailsNicely() {
                     "version=2.0\n");
         });
     }
+
+    @Test
+    public void test_default_output() throws HeaderParseException {
```

Comment:
should be `testDefaultOutput` for java code style

Reply:
Have you seen in other projects tests are usually in this snake case, I find it more readable since test names can get wordy. Either way no strong preference, I'll change to camelCase